FastQC/0.11.8/FastQC.nf

```diff
@@ -1,17 +1,19 @@
+params.fastqc.optional
 
 process FastQC {
     tag {"FastQC ${sample_id} - ${rg_id}"}
     label 'FASTQC_0_11_8'
+    container = 'quay.io/biocontainers/fastqc:0.11.8--1'
+    shell = ['/bin/bash', '-euo', 'pipefail']
 
     input:
-    tuple sample_id, rg_id, file(fastq: "*")
```

Comment:
Dit is denk ik niet echt handig. Je hebt namelijk niet altijd een tweede fastq file.